Jenkinsfile

```
@@ -46,6 +46,9 @@ pipeline {
     buildDiscarder(logRotator(numToKeepStr: '5')) //, artifactNumToKeepStr: '30'
     copyArtifactPermission('*');
   }
+  parameters {
+      string defaultValue: 'pipeline-master', description: '', name: 'EE_BRANCH_NAME'
```

We can add a description here. Maybe something like the following:
```suggestion
      string defaultValue: 'pipeline-master', description: 'The name of the EE branch to run the EE pipeline on', name: 'EE_BRANCH_NAME'
```